sdk/resourcemanager/Azure.ResourceManager/src/PsuedoGenerated/Resources/ResourceLinkContainer.cs

```diff
@@ -0,0 +1,388 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Core;
+using Azure.ResourceManager.Resources.Models;
+
+namespace Azure.ResourceManager.Resources
+{
+    /// <summary> A class representing collection of ResourceLink and their operations over a Tenant. </summary>
+    public partial class ResourceLinkContainer : ArmContainer
+    {
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly ResourceLinksRestOperations _restClient;
+
+        /// <summary> Initializes a new instance of the <see cref="ResourceLinkContainer"/> class for mocking. </summary>
+        protected ResourceLinkContainer()
+        {
+        }
+
+        /// <summary> Initializes a new instance of ResourceLinkContainer class. </summary>
+        /// <param name="parent"> The resource representing the parent resource. </param>
+        internal ResourceLinkContainer(ArmResource parent) : base(parent)
+        {
+            _clientDiagnostics = new ClientDiagnostics(ClientOptions);
+            _restClient = new ResourceLinksRestOperations(_clientDiagnostics, Pipeline, BaseUri);
+        }
+
+        /// <summary> Verify that the input resource Id is a valid container for this type. </summary>
+        /// <param name="identifier"> The input resource Id to check. </param>
+        protected override void ValidateResourceType(ResourceIdentifier identifier)
+        {
+        }
```

Comment:
@Yao725 as discussed, let's remove this method.

Reply:
removed